### PR TITLE
adxcgBidAdapter: updated backend protocol

### PR DESCRIPTION
**Pull request by adxcgcom**
## Type of change
- [X] Feature

## Description of change
 Change of backend integration protocol from internal to official openrtb 2.5

- test parameters for validating bids
```
{
 var adUnits = [{
                    code: 'banner-ad-div',
                    mediaTypes: {
                        banner: {
                            sizes: [
                                [300, 250],
                                [300, 600]
                            ]
                        }
                    },
                    bids: [{
                        bidder: 'adxcg',
                        params: {
                            adzoneid: '1'
                        }
                    }]
                }, {
                    code: 'native-ad-div',
                    mediaTypes: {
                        native: {
                             image: {
                                sendId: false,
                                required: false,
                                sizes: [127, 83]
                            },
                            icon: {
                                sizes: [80, 80],
                                required: false,
                                sendId: true,
                            },
                            title: {
                                sendId: false,
                                required: false,
                                len: 75
                            },
                            body: {
                                sendId: false,
                                required: false,
                                len: 200
                            },
                            cta: {
                                sendId: false,
                                required: false,
                                len: 75
                            },
                            sponsoredBy: {
                                sendId: false,
                                required: false
                            }
                        }
                    },
                    bids: [{
                            bidder: 'adxcg',
                            params: {
                                adzoneid: '2379'
                            }                        
                    }]
            },
            {
                code: 'video-div',
                mediaTypes: {
                    video: {
                            playerSize: [640, 480],
                            context: 'instream',
                            mimes: ['video/mp4'],
                            protocols: [2, 3, 5, 6, 8],
                            playback_method: ['auto_play_sound_off'],
			                maxduration: 100,
                            skip: 1
                        }
                },
                bids: [{
                    bidder: 'adxcg',
                    params: {
                        adzoneid: '20'                       
                    }a
                }]
            }
        ];
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer   info@adxcg.com
- [X] official adapter submission

### CODE REVIEW

File: modules/adxcgBidAdapter.js [hunk 1]
```diff
@@ -1,410 +1,363 @@
...
-    if (syncOptions.iframeEnabled) {
-      return [{
-        type: 'iframe',
-        url: 'https://cdn.adxcg.net/pb-sync.html' + params
-      }];
+  onBidWon: (bid) => {
+    // for native requests we put the nurl as an imp tracker, otherwise if the auction takes place on prebid server
+    // the server JS adapter puts the nurl in the adm as a tracking pixel and removes the attribute
+    if (bid.nurl) {
+      ajax(replaceAuctionPrice(bid.nurl, bid.originalCpm))
```
Comment (Fawke) [inline modules/adxcgBidAdapter.js:327]: I'm not sure if a bid adapter is allowed to make an Ajax request apart from an Auction request. Is there any way you can avoid it?

**Review by Fawke — CHANGES_REQUESTED** [review]
@adxcgcom Hi, this looks good, just a small observation on the ajax request that I saw on the `onBidWon` hook. I'm not sure if this is allowed.

Apart from that, I see that this adapter doesn't have a Gvlid yet. If you have a Gvlid and if you think that'll help, I'd advise to add that.

**Comment by adxcgcom** [comment]
Fawke, thx for review. We updated from ajax to triggerPixel, as this is most commonly used in bidAdapters.  GvlID is ok as is.

**Review by Fawke — APPROVED** [review]
LGTM

**Pull request merged**